.buildkite/pipeline.yml

```diff
@@ -12,27 +12,54 @@
 # permissions and limitations under the License.
 
 steps:
-  - label: ':go: go mod download'
-    command: 'go mod download'
 
-  # We use a "wait" step here, because Go's module logic freaks out when
-  # multiple go builds are downloading to the same cache.
-  - wait
-
-  - label: 'build'
-    command: 'make'
+  - label: ":docker: Build"
+    env:
+      DOCKER_BUILDKIT: "1"
+    command:
+      >
+        docker build \
+          --progress=plain \
+          --file tools/docker/Dockerfile \
+          --target firecracker-containerd-test-naive \
+          --tag localhost/firecracker-containerd-test-naive:${BUILDKITE_BUILD_NUMBER} .
 
-  - label: ':hammer: snapshotter tests'
-    commands:
-      - "cd snapshotter"
-      - "DISABLE_ROOT_TESTS='true' make test EXTRAGOARGS='-v -count=1'"
```

Comment:
I'd like to see the non-root tests added back.  I think it's important that we verify that it's possible to disable root-required tests and still get meaningful results in CI.  This would be running a container without `--privileged`, without all the volumes, without `--ipc host` and running as a non-root user.

Reply:
I see the original intention now, this was straightforward so I'll be updating with it.